notify_phase_end.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+"""
+This script is used by various workloads to notify DSI that a phase within a workload ended.
+The messages from this script are parsed by DSI's PortForwardedEventQueue in events.py.
+If profiling is enabled, this will trigger DSI to stop any profilers that are currently running.
+"""
+
+import socket
+from argparse import ArgumentParser
+import json
+
+parser = ArgumentParser()
+parser.add_argument("phase_id", type=int, help="Number of the phase that ended")
```

Comment:
I mean i assume i am too late to suggest this but seems like phases should have string names rather than numbers.

Reply:
(this is not a merge blocker)

Reply:
It's unfortunately not too late ;) I have [a PR up](https://github.com/10gen/dsi/pull/1510) that adjusts how DSI is using the phase numbers and updated this PR accordingly.

Patch showing that the new setup is working:
https://spruce.mongodb.com/version/659ebbc4306615778148ee1e